packages/schema/src/doc.ts

```diff
@@ -148,6 +148,19 @@ export class JupyterGISDoc
     });
   }
 
+  addLayersTreeItem(index: number, item: IJGISLayerItem) {
+    this.transact(() => {
+      this._layersTree.insert(index, [item]);
+    });
+  }
+
+  updateLayersTreeItem(index: number, item: IJGISLayerItem) {
+    this.transact(() => {
+      this._layersTree.delete(index);
+      this._layersTree.insert(index, [item]);
+    });
+  }
+
```

Comment:
The changes on the tree are propagated to collaborators, but not save in the file.
In a regular usage (new layer and layer included in tree), it works well because the new layer trigger the change on the server side.
But if we only include a layer in the tree, it is not saved in the file.
@davidbrochart any thoughts on this ?

Reply:
Is this still relevant or have you figured out your issue?

Reply:
No, I haven't figured it out.
It seems that it does not occur if the layers (and probably the sources, I didn't try) are updated, only the `layerTree`.

Reply:
Let's open an issue for this